memfault/memfault-docs
# MFLT-8016 Document dev-mode and new commands for Linux SDK 1.2.0

MFLT-8016 Document dev-mode and new commands for Linux SDK 1.2.0

  ### Summary

This documents the new SDK features included in Linux SDK 1.2.0.

  - memfaultctl and all it's new features

I have also split the getting started into:

  - A quickstart guide with the instructions to use the Docker image (this used
  to be only in a README in the meta-memfault-example layer) but I think it's
  very valuable and wanted to make this more visible.

  - An integration guide (used to be called getting started guide). This matches
  what we do on MCUs.

And I have renamed some pages to avoid having linux multiple times in the page
URL (again this matches what we do for MCUs and most Android pages).

For each renamed page I have added a redirect in the Netlify configuration.

  ### Test Plan

- Make sure the doc builds.
- Make sure all renamed pages have an entry in the `_redirects` file.
- Proof read for grammar and spelling.
- Make sure all links have been updated to the new URLs
